src/backends/plotly.jl

```diff
@@ -810,9 +810,14 @@ html_body(plt::Plot{PlotlyBackend}) = plotly_html_body(plt)
 const ijulia_initialized = Ref(false)
 
 function plotly_html_head(plt::Plot)
-    local_file = ("file://" * plotly_local_file_path)
+    local_file = ("file:///" * plotly_local_file_path)
     plotly =
         use_local_dependencies[] ? local_file : "https://cdn.plot.ly/plotly-latest.min.js"
+
+    include_mathjax = get(plt.attr, :include_mathjax, "")
```

Comment:
this should be `get(plt[:extra_plot_kwargs], :include_mathjax, "")`
Call site would be `plot(..., include_mathjax = "cdn", extra_kwargs = :plot)`.

Reply:
Done